### ISSUE TITLE
Allow additional information on the comment that approves/denies

### ISSUE THREAD

**Issue by Phoenix616** [trstringer/manual-approval#21]
Currently the messages have to stand in their own comment. It would be ideal to be able to to provide additional information on new lines after the keyword especially in the case of denials to provide some kind of reason in an easy way (instead of having to post two comments)

Also while I'm at that: #12 [was not fixed for denials](https://github.com/trstringer/manual-approval/blame/6b85d8fb556e097a84a2609af517dd6ccbb91f56/approval.go#L150).

**Comment by trstringer** [comment]
Thanks for opening up this issue. And wow good catch for #8 for denials! I'll fix that asap.

Regarding the original issue, I'm slightly inclined to continue the workflow of having multiple comments for additional context.

So you feel as though it would be better if this was a single comment?

```
denied.

Here is the reason that it was denied.
```

**Comment by Phoenix616** [comment]
> So you feel as though it would be better if this was a single comment?
> 
> ```
> denied.
> 
> Here is the reason that it was denied.
> ```

Yes. The reasoning behind this would be that with the current implementation one would have to pre-write the approval/denial reason somewhere and then copy it into another comment after posting the approval/denial comment if one would want the reasoning to be available instantly. Which imo. can be important as people might be pretty fast in checking especially if something was denied so you would want the reason to be there too as soon as possible.

**Comment by trstringer** [comment]
Good point. Maybe I should convert this into a slash command. So it would match on `/deny`, `/lgtm`, etc. What do you think? Would be easier to see "hey that's a command that will be handled" as opposed to just a normal string like current.

**Comment by Phoenix616** [comment]
Hm, I kinda like the distinction but if anything I think that should be optional or configurable. I could imagine that such a change could break quite some people's workflows/lead to delays until the change actually reached everyone who's working with it.

**Comment by trstringer** [comment]
Yeah I definitely can't break existing v1 users. I'd have to put the slash command behind a feature flag.

**Comment by alex-ld** [comment]
Reading this thread makes me think what if keywords for both approval and denial occur at the same time in the comment? Which one will take precedence?
